security-demo/cloud-stub/cmd/config/app.conf

```diff
@@ -0,0 +1 @@
+placementURL = "http://127.0.0.1:20000"
```

Comment:
I don't think placement is required in kubeedge.

Reply:
The demo is based on v0.1. So placement mock is required. In the next version, when cloud components are integrated, all config and changes for cloud stub will be removed.

Reply:
This will be fixed when we upgrade to v0.2.